tests/functional/pointerevent_pointerleave_descendants-manual.js

```diff
@@ -0,0 +1,20 @@
+define(function(require) {
+	var registerSuite = require('intern!object');
+	var w3cTest = require('../support/w3cTest');
+	var name = 'pointerevent_pointerleave_descendants-manual';
+
+	registerSuite({
+		name: name,
+
+		main: function() {
+			return w3cTest(this.remote, name + '.html')
+				.findById('target0')
+					.moveMouseTo(50, 8)
+					.end()
+				.findByTagName('body')
+					.moveMouseTo(50, 30)
```

Comment:
30px from the top of the body should be above the black box, not below the purple box. I'm not sure how this would work. Am I not understanding how the values work for `.moveMouseTo()`?